contracts/multisig/src/multisig.rs

```diff
@@ -117,19 +119,39 @@ pub trait Multisig:
                 }
             }
         }
+
         result
     }
 
     /// Clears storage pertaining to an action that is no longer supposed to be executed.
     /// Any signatures that the action received must first be removed, via `unsign`.
     /// Otherwise this endpoint would be prone to abuse.
     #[endpoint(discardAction)]
-    fn discard_action(&self, action_id: ActionId) {
+    fn discard_action_endpoint(&self, action_id: ActionId) {
         let (_, caller_role) = self.get_caller_id_and_role();
         require!(
             caller_role.can_discard_action(),
             "only board members and proposers can discard actions"
         );
+
+        self.discard_action(action_id);
+    }
+
+    /// Discard all the actions in the given group
+    #[endpoint(discardBatch)]
+    fn discard_batch(&self, group_id: GroupId) {
```

Comment:
I would keep the model used in signing batches, using action ids:
`fn sign_batch(&self, action_ids: MultiValueEncoded<ActionId>)`
Let the front-end decide to group ids by batches.

Reply:
Done.